lib/fs.js

```diff
@@ -167,7 +167,7 @@ function nullCheck(path, callback) {
 }
 
 function isFd(path) {
-  return (path >>> 0) === path;
+  return (typeof path === 'number' && ~~path === path);
```

Comment:
Maybe leave a comment on that? I'm not sure how many people understand what `~~` does here.